controllers/testdata/appconfig-setters-expected/deploy.yaml

```diff
@@ -7,4 +7,4 @@ spec:
     spec:
       containers:
         - name: hello
-          image: helloworld:1.0.1 # SETTER_SITE
+          image: helloworld:v1.3.1 # SETTER_SITE
```

Comment:
Yes, it was deliberate or incidentally add to adapt to the new structure because the subtests shared the same image policy.
I just tried moving the image policy from being common to all the subtests, to be created per subtests. That helps avoid changing the above testdata and removes the dependence on the ordering of the tests. Much better now.
Thanks for emphasizing this.